src/Service/Bulk/BulkCreateUsersService.php

```diff
@@ -105,4 +117,16 @@ public function generate(
             }
         }
     }
+
+    private function filterLineItems(array $lineItems): array
```

Comment:
In case if function expects a param of array type if would be helpful to add a phpdoc with defining a type of array items
    
```suggestion
    /**
    * @param LineItem[] $lineItems
    */
    private function filterLineItems(array $lineItems): array
```